bindings/gumjs/gumdukcore.c

```diff
@@ -2782,7 +2780,7 @@ gumjs_native_function_init (duk_context * ctx,
 unexpected_marker:
   {
     gum_duk_native_function_finalize (func);
-    _gum_duk_throw (ctx, "only one variadic marker may be specified");
+    _gum_duk_throw (ctx, "only one variadic marker may be specified and can not be first argument");
```

Comment:
Exceeds 80 cols. Should be wrapped using this style:

```suggestion
    _gum_duk_throw (ctx, "only one variadic marker may be specified and can "
        "not be first argument");
```